message-service/src/main/java/sjchat/messages/MessageService.java

```diff
@@ -56,52 +79,51 @@ public void getChatList(GetChatListRequest req, StreamObserver<GetChatListRespon
 
   @Override
   public void getChat(GetChatRequest req, StreamObserver<GetChatResponse> responseObserver) {
-    User.Builder userBuilder1 = buildMockUser("user-1");
-    User.Builder userBuilder2 = buildMockUser("user-2");
-
-    Chat.Builder chatBuilder1 = buildMockChat("chat-1");
-    chatBuilder1.addParticipants(userBuilder1);
-    chatBuilder1.addParticipants(userBuilder2);
+    ChatEntity entity = chatDao.find(req.getId());
+    GetChatResponse chatResponse;
+    if(entity == null){
+      chatResponse = GetChatResponse.newBuilder().setChat(Chat.newBuilder().setId("null").setTitle("Not existing")).build();
+    }else{
+      chatResponse = GetChatResponse.newBuilder().setChat(buildChat(entity)).build();
+    }
 
-    GetChatResponse chatResponse = GetChatResponse.newBuilder().setChat(chatBuilder1).build();
 
     responseObserver.onNext(chatResponse);
     responseObserver.onCompleted();
   }
 
   @Override
   public void createChat(CreateChatRequest req, StreamObserver<CreateChatResponse> responseObserver) {
-    Random random = new Random();
-
-    Chat.Builder chatBuilder = Chat.newBuilder();
-    chatBuilder.setId("mock");
+    ChatEntity.Builder chatBuilder = new ChatEntity.Builder();
+    chatBuilder.setId(null);
     chatBuilder.setTitle(req.getTitle());
-    for (String userId : req.getParticipantsList()) {
-      User.Builder userBuilder = User.newBuilder();
-      userBuilder.setId(userId);
-      userBuilder.setUsername("mock_username");
-      chatBuilder.addParticipants(userBuilder);
-    }
+    //TODO: Check that all participants are actual valid userids
```

Comment:
Please create an issue instead of //TODO flags. This way it's easier for other people on the team to pick tasks they can work on too.